standalone/api.c

```diff
@@ -790,18 +790,16 @@ void modsecReportRemoteLoadedRules()
 #ifdef WAF_JSON_LOGGING_ENABLE
 void modsecReopenLogfileIfNeeded(request_rec *r)
 {
-    modsec_rec *msr = NULL;
     int rc = 0;
     apr_file_t * fd = NULL;
     /* Find the transaction context first. */
```

Comment:
This comment is now outdated and therefore misleading. Let's remove it altogether with the code retrieving `msr`

Reply:
Good to see u again, Vlad, will change